packages/lambda-powertools-middleware-log-timeout/index.js

```diff
@@ -0,0 +1,42 @@
+const Promise = require('bluebird')
+const Log = require('@perform/lambda-powertools-logger')
+
+module.exports = (thresholdMillis = 10) => {
+  let isTimedOut
+  let promise
```

Comment:
I think it'd be safer and more understandable to assign to the `context` rather than using globals and resetting them for every invocation

Reply:
I think it's a good idea, but now that it's open source, I do worry adding more and more things to the `context` object as we can accidentally override properties others put in when there's a name clash. We've not done this so far, but we might need to consider scoping all the things we're adding to the powertools to specific properties.